Add `Gameplay/Hit window` (#14591)
ppy/osu-wiki: wiki/Gameplay/Hit_window/en.md

# Hit window

> ![](img/hit_window.png "Exemplary scheme of a hit window visualising the possible outcomes of a hit")

Reviewer: missing alt text

```suggestion
![A hit window graph](img/hit_window.png "A hit window graph visualising the possible outcomes of a hit")
```

> Each [hit object](/wiki/Gameplay/Hit_object) has a time frame called **hit window** in which players are required to hit them. The higher the accuracy of the hit, the more score will be rewarded. Failing to hit the hit object at all will incur a miss.

Reviewer: ```suggestion
Each [hit object](/wiki/Gameplay/Hit_object) has a time frame called a **hit window** in which players are required to hit them. The higher the accuracy of the hit, the more score will be rewarded. Failing to hit the hit object at all will incur a miss.
```

> The length of a hit window is affected by the [overall difficulty](/wiki/Beatmap/Overall_difficulty) as well as rate-changing mods like [Double Time](/wiki/Gameplay/Game_modifier/Double_Time) and [Half Time](/wiki/Gameplay/Game_modifier/Half_Time).

Reviewer: ```suggestion
The regions within a hit window award different [judgements](/wiki/Gameplay/Judgement) and can each be considered a nested hit window. Their lengths are affected by the [overall difficulty](/wiki/Beatmap/Overall_difficulty) as well as rate-changing mods like [Double Time](/wiki/Gameplay/Game_modifier/Double_Time) and [Half Time](/wiki/Gameplay/Game_modifier/Half_Time).
```

> osu!catch has no concept of timing, so any hit objects will be judged as either hit or missed.

Author: ```suggestion
[osu!catch](/wiki/Game_mode/osu!catch) has no concept of timing, so any hit objects will be judged as either hit or missed.
```

> [osu!catch](/wiki/Game_mode/osu!catch) has no concept of timing, so any hit objects will be judged as either hit or missed.

Reviewer: ```suggestion
The amount and length of hit windows also varies by [game mode](/wiki/Game_mode). For example, [osu!catch](/wiki/Game_mode/osu!catch) has no concept of timing, so any hit objects will be judged as either hit or missed.
```
